### ISSUE TITLE
Deleting workspace feed from Set Commercial Card Feed supportal tool doesn't actually delete it.

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#54377]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:**
**Reproducible in production?:**
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @joekaufmanexpensify 
**Slack conversation** (hyperlinked to channel name): [expense](https://expensify.slack.com/archives/C06ML6X0W9L/p1734560618043959)

## Action Performed:
Prerequisite: User added a company card

1. Login via supportal and delete the added custom feed card

## Expected Result:
The Commercial card feed deleted

## Actual Result:
The card feed is removed in the supportal page but not on the Newdot even after refreshing

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

https://github.com/user-attachments/assets/94f6104f-a8ec-42ee-b6c1-19984beea7bf

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mountiny** [comment]
havent got around yet

**Comment by mountiny** [comment]
Havent got around yet

**Comment by mountiny** [comment]
Discussing with @robertjchen and @nkuoch about the expected behaviour here as the command used from the supportal now is not doing the same things as the command used when user removes their own feed

**Comment by greg-schroeder** [comment]
Thanks @mountiny - is there a link to that discussion so I can follow?

**Comment by mountiny** [comment]
Discussed in slack DM. Robert and others agreed there is not much reason to make these commands different.

So the next step is to make sure the supportal tool works the same as removing feed in from the product, I will try to tackle it this week

**Comment by mountiny** [comment]
On my list, but focusing on fire clean up

**Comment by greg-schroeder** [comment]
Thanks @mountiny <3

**Comment by mountiny** [comment]
Still on my list

**Comment by greg-schroeder** [comment]
I wouldn't want to be on Vit's list, lemme tell ya

**Comment by mountiny** [comment]
Discussed with Nathalie - I will update the RemoveFeed command to work well with the supportal tool and use that. Then if the existing command `RemoveCommercialFeed` is not used elsewhere we can remove it

**Comment by greg-schroeder** [comment]
Still on Vit's list

**Comment by mountiny** [comment]
Okay, you know, Greg. I get it. It is still on my list, and it's been sitting patiently, but enough is enough. I will make you proud of me

**Comment by mountiny** [comment]
Stability issue is taking higher priority now

**Comment by mountiny** [comment]
Started a draft for Auth PR

**Comment by mountiny** [comment]
Work in progress

**Comment by mountiny** [comment]
Still WIP

**Comment by greg-schroeder** [comment]
Thanks @mountiny

**Comment by greg-schroeder** [comment]
WIP

**Comment by mountiny** [comment]
After working more on the PR, unfortunately I think I will take a slightly different approach. Adding the supportal functionality to the RemoveFeed command feels like its complicating it too much unnecessarily so I think also from the 1:1:1 philosophy in general it will be better to update the existing supportal command

**Comment by greg-schroeder** [comment]
Thanks for the update @mountiny

**Comment by mountiny** [comment]
Focusing on `stability` issue

**Comment by greg-schroeder** [comment]
Still on that #stability item Vit?

**Comment by mountiny** [comment]
Yes, its my main focus plus critical projects in `migrate`

**Comment by greg-schroeder** [comment]
Vit is still working on #migrate initiatives

**Comment by mountiny** [comment]
Yeah, mainly the stability issue is taking my time but would aim to finish this one up this week

**Comment by greg-schroeder** [comment]
Okay thanks @mountiny!

**Comment by mountiny** [comment]
Got hit with some virus reducing my working hours

**Comment by mountiny** [comment]
PR in review

**Comment by mountiny** [comment]
We have just one small follow up to do here, but coming back from ooo, will handle shortly

**Comment by mountiny** [comment]
this should be done when @joekaufmanexpensify was on vacay. @joekaufmanexpensify would you be able to retest if the Supportal tool also correctly closes the cards? Thank you!

**Comment by joekaufmanexpensify** [comment]
Tested and confirmed it works. Thanks for fixing this!
